plugins/verifier/schemavalidator/README.md

```diff
@@ -20,7 +20,7 @@ Schemas can be configured in Ratify config.json or via CRD.
 apiVersion: config.ratify.deislabs.io/v1beta1
 kind: Verifier
 metadata:
-  name: verifier-schemavalidator
+  name: schemavalidator
```

Comment:
general comment: would it be possible to comment on PR why the naming convention changed? I see most of the samples have had similar changes.